truffle.js

```diff
@@ -0,0 +1,53 @@
+require('dotenv').config()
+const HDWalletProvider = require('truffle-hdwallet-provider')
+
+module.exports = {
+  compilers: {
+    solc: {
+      version: process.env.SOLC_VERSION || '0.4.25',
+      settings: {
+        optimizer: {
+          enabled: true,
+          runs: 1
+        }
+      }
+    }
+  },
+  networks: {
+    development: {
```

Comment:
This would override the behavior for running `truffle test`. Without `development` it would automatically spin up a blockchain for you and with this we need to spin up the blockchain ourselves that matches the configuration here.

Do we really need to config this now?

Reply:
Ah ok. I think it's useful for deploying to a local eth network.
I'll change the name of it to `local` to avoid overriding `truffle test` behaviour